test/unit/api/define.js

```diff
@@ -3,18 +3,38 @@ import { Component, define, symbols } from '../../../src/index';
 const { name: $name } = symbols;
 
 describe('api/define', () => {
+  it('should not register without any properties', () => {
+    expect(() => {
+      define('x-test');
+    }).to.throw(Error);
+  });
+
+  it('should not register a component without a dash in the name', () => {
```

Comment:
If we're going to test this, we would want to test [allthethings](http://w3c.github.io/webcomponents/spec/custom/#valid-custom-element-name) to be consistent, but I don't think it's worth it if we don't have anything to do with it.